### PR TITLE
Fix integration test failure to parse eksctl version

### PR DESCRIPTION
**Pull request by michaelbeaumont**
### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [ ] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

File: pkg/managed/service.go [hunk 1]
```diff
@@ -255,15 +255,15 @@ func (m *Service) requiresStackUpdate(nodeGroupName string) (bool, error) {
...
+	ver, found, err := manager.GetEksctlVersionFromTags(ngStack.Tags)
 	if err != nil {
 		return false, err
 	}
 	if !found {
 		return true, nil
 	}
 
-	curVer, err := semver.ParseTolerant(version.GetVersion())
+	curVer, err := version.ParseEksctlVersion(version.GetVersion())
```
Comment (cPu1) [inline pkg/managed/service.go:266]: `ParseEksctlVersion` is not defined. Did you mean `manager.GetEksctlVersionFromTags`? 

```suggestion
	curVer, err := manager.GetEksctlVersionFromTags(version.GetVersion())
```

Comment (cPu1) [inline pkg/managed/service.go:266]: `ParseEksctlVersion` is not defined. Did you mean to change this line?

Comment (michaelbeaumont) [reply pkg/managed/service.go:266]: Sorry, forgot to `git add` the file.

**Review by cPu1 — APPROVED** [review]

**Comment by michaelbeaumont** [comment]
Fixed the tests

**Pull request merged**